src/app/frontend/replicationcontrollerdetail/replicationcontrollerdetail.html

```diff
@@ -40,7 +42,9 @@ <h1 flex class="md-title kd-replicationcontrollerdetail-app-name">
       <div flex layout="column" class="kd-replicationcontrollerdetail-sidebar-info">
         <span class="kd-replicationcontrollerdetail-sidebar-line">Name</span>
         <span class="kd-replicationcontrollerdetail-sidebar-subline">
-          {{::ctrl.replicationControllerDetail.name}}
+          <kd-middle-ellipsis display-string="{{::ctrl.replicationControllerDetail.name}}"
+                              max-length="{{::ctrl.nameMaxLength}}">
```

Comment:
I've adjusted this to widest possible name I could find that contained only `m` letters. We are not using monospaced font so all letters have different width.